src/transformer/encode.ts

```diff
@@ -253,6 +256,24 @@ const childElements = (node: ManuscriptNode): ManuscriptNode[] => {
 
   return nodes
 }
+const sectionChildElements = (node: ManuscriptNode): ManuscriptNode[] => {
+  const nodes: ManuscriptNode[] = []
+
+  node.forEach((childNode) => {
+    if (
+      !(
+        isSectionNode(childNode) ||
+        isBibliographySectionNode(childNode) ||
+        isGraphicalAbstractSectionNode(childNode) ||
+        isFootnotesSectionNode(childNode)
```

Comment:
Can we check the group of the section category instead of checking the types one by one?

Reply:
Done